### ISSUE TITLE
"Error: Can't resolve '@swc/helpers/" when upgrading from 2.14.0 to 2.17.0

### ISSUE THREAD

**Issue by pavinduLakshan** [TexteaInc/json-viewer#308]
I'm upgrading json-viewer from 2.14.0 to 2.17.0 in my react app. After upgrading, the following errors appear when running my react application.

```
ERROR in ./node_modules/@textea/json-viewer/dist/index.mjs 2:0-59
Module not found: Error: Can't resolve '@swc/helpers/_/_define_property' in '/Users/pavindu/Downloads/asgardeo-react-app 3/node_modules/@textea/json-viewer/dist'
 @ ./src/components/authentication-response.tsx 20:0-49 45:38-48 71:152-162 81:153-163
 @ ./src/components/index.tsx 19:0-42 19:0-42
 @ ./src/pages/home.tsx 34:0-55 156:38-60
 @ ./src/pages/index.tsx 20:0-23 20:0-23
 @ ./src/app.tsx 26:0-49 34:46-54 36:46-58

ERROR in ./node_modules/@textea/json-viewer/dist/index.mjs 3:0-57
Module not found: Error: Can't resolve '@swc/helpers/_/_object_spread' in '/Users/pavindu/Downloads/asgardeo-react-app 3/node_modules/@textea/json-viewer/dist'
 @ ./src/components/authentication-response.tsx 20:0-49 45:38-48 71:152-162 81:153-163
 @ ./src/components/index.tsx 19:0-42 19:0-42
 @ ./src/pages/home.tsx 34:0-55 156:38-60
 @ ./src/pages/index.tsx 20:0-23 20:0-23
 @ ./src/app.tsx 26:0-49 34:46-54 36:46-58

ERROR in ./node_modules/@textea/json-viewer/dist/index.mjs 4:0-56
Module not found: Error: Can't resolve '@swc/helpers/_/_object_spread_props' in '/Users/pavindu/Downloads/asgardeo-react-app 3/node_modules/@textea/json-viewer/dist'
 @ ./src/components/authentication-response.tsx 20:0-49 45:38-48 71:152-162 81:153-163
 @ ./src/components/index.tsx 19:0-42 19:0-42
 @ ./src/pages/home.tsx 34:0-55 156:38-60
 @ ./src/pages/index.tsx 20:0-23 20:0-23
 @ ./src/app.tsx 26:0-49 34:46-54 36:46-58

ERROR in ./node_modules/@textea/json-viewer/dist/index.mjs 5:0-63
Module not found: Error: Can't resolve '@swc/helpers/_/_to_consumable_array' in '/Users/pavindu/Downloads/asgardeo-react-app 3/node_modules/@textea/json-viewer/dist'
 @ ./src/components/authentication-response.tsx 20:0-49 45:38-48 71:152-162 81:153-163
 @ ./src/components/index.tsx 19:0-42 19:0-42
 @ ./src/pages/home.tsx 34:0-55 156:38-60
 @ ./src/pages/index.tsx 20:0-23 20:0-23
 @ ./src/app.tsx 26:0-49 34:46-54 36:46-58

ERROR in ./node_modules/@textea/json-viewer/dist/index.mjs 9:0-59
Module not found: Error: Can't resolve '@swc/helpers/_/_sliced_to_array' in '/Users/pavindu/Downloads/asgardeo-react-app 3/node_modules/@textea/json-viewer/dist'
 @ ./src/components/authentication-response.tsx 20:0-49 45:38-48 71:152-162 81:153-163
 @ ./src/components/index.tsx 19:0-42 19:0-42
 @ ./src/pages/home.tsx 34:0-55 156:38-60
 @ ./src/pages/index.tsx 20:0-23 20:0-23
 @ ./src/app.tsx 26:0-49 34:46-54 36:46-58

ERROR in ./node_modules/@textea/json-viewer/dist/index.mjs 10:0-51
Module not found: Error: Can't resolve '@swc/helpers/_/_type_of' in '/Users/pavindu/Downloads/asgardeo-react-app 3/node_modules/@textea/json-viewer/dist'
 @ ./src/components/authentication-response.tsx 20:0-49 45:38-48 71:152-162 81:153-163
 @ ./src/components/index.tsx 19:0-42 19:0-42
 @ ./src/pages/home.tsx 34:0-55 156:38-60
 @ ./src/pages/index.tsx 20:0-23 20:0-23
 @ ./src/app.tsx 26:0-49 34:46-54 36:46-58

ERROR in ./node_modules/@textea/json-viewer/dist/index.mjs 12:0-62
Module not found: Error: Can't resolve '@swc/helpers/_/_async_to_generator' in '/Users/pavindu/Downloads/asgardeo-react-app 3/node_modules/@textea/json-viewer/dist'
 @ ./src/components/authentication-response.tsx 20:0-49 45:38-48 71:152-162 81:153-163
 @ ./src/components/index.tsx 19:0-42 19:0-42
 @ ./src/pages/home.tsx 34:0-55 156:38-60
 @ ./src/pages/index.tsx 20:0-23 20:0-23
 @ ./src/app.tsx 26:0-49 34:46-54 36:46-58

ERROR in ./node_modules/@textea/json-viewer/dist/index.mjs 13:0-56
Module not found: Error: Can't resolve '@swc/helpers/_/_ts_generator' in '/Users/pavindu/Downloads/asgardeo-react-app 3/node_modules/@textea/json-viewer/dist'
 @ ./src/components/authentication-response.tsx 20:0-49 45:38-48 71:152-162 81:153-163
 @ ./src/components/index.tsx 19:0-42 19:0-42
 @ ./src/pages/home.tsx 34:0-55 156:38-60
 @ ./src/pages/index.tsx 20:0-23 20:0-23
 @ ./src/app.tsx 26:0-49 34:46-54 36:46-58

ERROR in ./node_modules/@textea/json-viewer/dist/index.mjs 14:0-54
Module not found: Error: Can't resolve '@swc/helpers/_/_instanceof' in '/Users/pavindu/Downloads/asgardeo-react-app 3/node_modules/@textea/json-viewer/dist'
 @ ./src/components/authentication-response.tsx 20:0-49 45:38-48 71:152-162 81:153-163
 @ ./src/components/index.tsx 19:0-42 19:0-42
 @ ./src/pages/home.tsx 34:0-55 156:38-60
 @ ./src/pages/index.tsx 20:0-23 20:0-23
 @ ./src/app.tsx 26:0-49 34:46-54 36:46-58

ERROR in ./node_modules/@textea/json-viewer/dist/index.mjs 15:0-52
Module not found: Error: Can't resolve '@swc/helpers/_/_to_array' in '/Users/pavindu/Downloads/asgardeo-react-app 3/node_modules/@textea/json-viewer/dist'
 @ ./src/components/authentication-response.tsx 20:0-49 45:38-48 71:152-162 81:153-163
 @ ./src/components/index.tsx 19:0-42 19:0-42
 @ ./src/pages/home.tsx 34:0-55 156:38-60
 @ ./src/pages/index.tsx 20:0-23 20:0-23
 @ ./src/app.tsx 26:0-49 34:46-54 36:46-58

ERROR in ./node_modules/@textea/json-viewer/dist/index.mjs 17:0-69
Module not found: Error: Can't resolve '@swc/helpers/_/_object_without_properties' in '/Users/pavindu/Downloads/asgard
```

It'd be great if someone can point out what has gone wrong? we have been using the JsonViewer in our project as follows.

```jsx
import { JsonViewer } from '@textea/json-viewer'

 <JsonViewer
      className="asg-json-viewer"
      value={ derivedResponse?.authenticateResponse }
      enableClipboard={ false }
      displayObjectSize={ false }
      displayDataTypes={ false }
      rootName={ false }
      theme="dark" 
/>
```

**Comment by gfazioli** [comment]
I got the same 
```
./node_modules/.pnpm/@textea+json-viewer@2.17.0_@types+react@18.0.28_react-dom@18.2.0_react@18.2.0/node_modules/@textea/json-viewer/dist/index.mjs
Module not found: Can't resolve '@swc/helpers/_/_define_property'
```

**Comment by pavinduLakshan** [comment]
I tried to upgrade to 2.16.2 and these errors didn't appear. Seems like something has gone wrong with v2.17.0.   

cc: @pionxzh

**Comment by pionxzh** [comment]
I can reproduce this on the stackblitz demo. Not sure if #302 introduced something untended.

**Comment by pionxzh** [comment]
Fixed in v2.17.1

**Comment by pavinduLakshan** [comment]
Thanks @pionxzh for the prompt response and action 🙏
